phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexUpgradeTool.java

```diff
@@ -103,6 +103,11 @@
             "[Optional]Whether or not synchronously rebuild the indexes; "
                     + "default rebuild asynchronous");
 
+    private static final Option INDEX_VERIFY_OPTION = new Option("v",
+            "verify",
+            true,
+            "[Optional] mode to run indexTool with verify options [BEFORE|AFTER|NONE|ONLY|BOTH]");
```

Comment:
For discussion: I would have preferred not to add this list here since we would every time we add new options in IndexTool, we would have to keep adapting IndexUpgradeTool. Not a big ask, but something that the community has to remember to do each time. Alternative is to keep these opaque and not even list here.